pkg/core/native/native_neo.go

```diff
@@ -1021,7 +1071,7 @@ func (n *NEO) computeCommitteeMembers(blockHeight uint32, d *dao.Simple) (keys.P
 	count := n.cfg.GetCommitteeSize(blockHeight + 1)
 	// Can be sorted and/or returned to outside users, thus needs to be copied.
 	sbVals := keys.PublicKeys(n.standbyKeys[:count]).Copy()
-	cs, err := n.getCandidates(d, false)
+	cs, err := n.getCandidates(d, false, -1)
```

Comment:
Well, we only need `count` number of members then.

Reply:
Yes, but the problem is that they should not be sorted by key bytes. They should be sorted by votes, and in this case we still need to retrieve the whole set of candidates.